### PR TITLE
EPMRPP-91158 || Updates according to the new async reporting approach

### PR DESCRIPTION
**Pull request by pbortnik**

### CODE REVIEW

File: docs/developers-guides/AsynchronousReporting.mdx [hunk 1]
```diff
@@ -63,19 +65,19 @@ rp.reporting.async=true
...
-* [Start child(container) item](/developers-guides/ReportingDevelopersGuide#start-childcontainer-item)
-* [Start child(step) item](/developers-guides/ReportingDevelopersGuide#start-childstep-item)
-* [Finish child item](/developers-guides/ReportingDevelopersGuide#finish-child-item)
-* [Finish parent(container) item](/developers-guides/ReportingDevelopersGuide#finish-parentcontainer-item)
-* [Save single log without attachment](/developers-guides/ReportingDevelopersGuide#save-single-log-without-attachment)
-* [Batch save logs](/developers-guides/ReportingDevelopersGuide#batch-save-logs)
-* [Save launch log](/developers-guides/ReportingDevelopersGuide#save-launch-log)
-* [Finish root(suite) item](/developers-guides/ReportingDevelopersGuide#finish-rootsuite-item)
-* [Finish launch](/developers-guides/ReportingDevelopersGuide#finish-launch)
+[reporting guide](ReportingDevelopersGuide.md).
```
Comment (Vadim73i) [inline docs/developers-guides/AsynchronousReporting.mdx:68]: Why did you need to change the links?

**Comment by Vadim73i** [comment]
If images with -old.png are not used, can they be deleted?

**Review by Vadim73i — APPROVED** [review]

**Comment by AmsterGet** [comment]
May be retarget this PR to another branch, f.e. `RP-24.2` instead of `develop`?
As we shouldn't publish these docs until the 24.2 product release.

**Review by AmsterGet — APPROVED** [review]
